espanol/transcender/destino.md

```diff
@@ -0,0 +1,5 @@
+Justo antes de ver a Beatito había estado con Magola. El medio de una guerra no era el mejor momento para probar hongos alucinogénos por primera vez.
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.